src/views/catalog/CreateFromInstanceTypes/utils/utils.ts

```diff
@@ -4,7 +4,11 @@ import { adjectives, animals, uniqueNamesGenerator } from 'unique-names-generato
 import DataSourceModel from '@kubevirt-ui/kubevirt-api/console/models/DataSourceModel';
 import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
 import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
-import { convertUserDataObjectToYAML } from '@kubevirt-utils/components/CloudinitModal/utils/cloudinit-utils';
+import {
```

Comment:
Do we really need to keep the first line of this file `/* eslint-disable perfectionist/sort-objects */` ?